src/cmd/kssh/kssh.go

```diff
@@ -141,19 +146,19 @@ const (
 	SSH
 )
 
-// Returns botname, remaining arguments, action, error
+// Returns botName, remaining arguments, action, error
 // If the argument requires exiting after processing, it will call os.Exit
 func handleArgs(args []string) (string, []string, Action, error) {
 	remaining, found, err := kssh.ParseArgs(args, cliArguments)
 	if err != nil {
 		return "", nil, 0, fmt.Errorf("Failed to parse provided arguments: %v", err)
 	}
 
-	team := ""
+	botName := ""
```

Comment:
this was likely a result of a change made a while back when we realized that the kssh-er specifying the team was strictly worse than specifying the bot. couple of things must have been missed in the change-over. but yeah - that's why this mistake is here.